### PR TITLE
Fix to update arg collection

### PR DESCRIPTION
**Pull request by gautamarora**

### CODE REVIEW

File: __checks__/dev.test.js [hunk 1]
```diff
@@ -1,15 +1,14 @@
 const fs = require("fs");
-const { argv } = require("yargs");
+const { path, org } = require('../src/argv');
 
 describe("Development", () => {
-  let { path: rootPath, org } = argv;
-  let pkg = JSON.parse(fs.readFileSync(`${rootPath}/package.json`, "utf8"));
+  let pkg = JSON.parse(fs.readFileSync(`${path}/package.json`, "utf8"));
```
Comment (pgoldrbx) [inline __checks__/dev.test.js]: move this into a `beforeAll` hook to ensure it runs at test time rather than require-time

File: src/argv.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  .option('path', {
+    default: '.',
+    describe: 'your project directory'
+  })
+  .option('org', {
+    describe: 'your npm org'
+  })
+  .usage("Usage: $0 [path] --o [org]");
+
+module.exports = argv
```
Comment (pgoldrbx) [inline src/argv.js]: lint

File: __checks__/dev.test.js (hunk 1 above)
Comment (pgoldrbx) [reply __checks__/dev.test.js]: oh, sorry, it's unrelated to your change

File: src/argv.js (hunk 2 above)
Comment (gautamarora) [reply src/argv.js]: fixed :)

**Pull request merged**